File: Front/task-management/src/app/models/task.ts
export enum Priority {
    Usual,
    Important,
    Critical,
}

export class Task {
    isNeedNotify!: Boolean
    priority!: Priority
    id!: string
    name!: string
    description!: string
    userEmail!: string
    startDate!: Date
    finishDate!: Date
}

File: Back/TaskManagement/TaskManagement.EmailService/Models/Task.cs
﻿using System;

namespace TaskManagement.EmailService.Models
{
    public enum Priority
    {
        Usual,
        Important,
        Critical,
    }

    public class Task
    {
        public bool IsNeedNotify { get; set; }
        public string Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public string UserEmail { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime FinishDate { get; set; }
        public Priority Priority { get; set; }
    }
}
